### PR TITLE
feat: reply with components needed for xpub

### PR DESCRIPTION
**Pull request by obiyankenobi**

### CODE REVIEW

**Review by jansegre — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -31,7 +31,7 @@ ICONNAME   = nanos_app_hathor.gif
 APPVERSION = 0.0.1
 
 # The --path argument here restricts which BIP32 paths the app is allowed to derive.
-APP_LOAD_PARAMS = --appFlags 0x40 --path "44'/93'" --curve secp256k1 --curve ed25519 $(COMMON_LOAD_PARAMS)
+APP_LOAD_PARAMS = --appFlags 0x40 --path "44'/280'" --curve secp256k1 $(COMMON_LOAD_PARAMS)
```
Comment (msbrogli) [inline Makefile:35]: Where is it used? I think I've seen the path in a different file. I'm just concerned about duplicate values (so, multiple places to update if needed).

File: src/getXPub.c [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+// This is the button handler for the approval screen
+static unsigned int ui_getXPub_approve_button(unsigned int button_mask, unsigned int button_mask_counter) {
+    // tx is the offset within G_io_apdu_buffer
+    uint16_t tx = 0;
+    cx_ecfp_public_key_t public_key;
+    cx_ecfp_private_key_t private_key;
+    unsigned char chain_code[32];
+    unsigned char hash_160[20];
+    // bip32 path for 44'/280'/0'/0
+    uint32_t path[] = {44 | 0x80000000, 280 | 0x80000000, 0 | 0x80000000, 0};
```
Comment (msbrogli) [inline src/getXPub.c:44]: Shouldn't we copy as we did [here](https://github.com/HathorNetwork/ledger-app-hathor/pull/4/files#diff-1dba452b06ff9b969864abb8ad184181R17)?

File: Makefile (hunk 1 above)
Comment (obiyankenobi) [reply Makefile:35]: It's used when we derive the keys, but in those cases we have the array, not the string.

File: src/getXPub.c (hunk 2 above)
Comment (obiyankenobi) [reply src/getXPub.c:44]: It's already been changed in the other PR. I noticed that I was repeating it so I defined a variable with our bip44 path and always use it when deriving addresses.

But in this PR, as it was only this instance, it was "hardcoded".

**Review by pedroferreira1 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (obiyankenobi) [reply Makefile:35]: I also changed this on the other PR(#4), where the bip44 path is on a single file. `280` is now defined in the Makefile only.

**Review by msbrogli — APPROVED** [review]

**Pull request merged**